packages/slate/src/changes/with-schema.js

```diff
@@ -146,12 +118,14 @@ function normalizeNode(change, node, schema) {
     if (!normalize) return
 
     // Run the `normalize` function to fix the node.
+    let path = c.value.document.getPath(n.key)
     normalize(c)
 
     // Re-find the node reference, in case it was updated. If the node no longer
     // exists, we're done for this branch.
-    n = refindNode(c, n)
+    n = c.value.document.refindNode(path, n.key)
     if (!n) return
+    path = c.value.document.refindPath(path, n.key)
```

Comment:
Can you add a line break after this early return to match the rest of the code?

Reply:
Fixed. Thank you